app/clparamiko.py

```diff
@@ -0,0 +1,40 @@
+import paramiko
+import time
+from contextlib import contextmanager
+
+from args import Arguments
+from shell import Shell
+
+class ClParamiko(Arguments, Shell):
```

Comment:
@GitSergeyGit why `Cl` is used as a prefix? It is no more a client, right?
Please not one thing - we've renamed base class and now all the inheritors needs to change its names, which is no good.
How can we deal with this situation?

Reply:
I use Cl as a prefix abbreviated from Class Paramiko that there would be no name conflicts with the library Paramiko.